code/tests/integration/ui/cypress/support/e2e.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
I see that others have already mentioned these commented-out files, but seems like we can disable from the config the importing of these files if we're not using them